Rubberduck.VBEEditor/SafeComWrappers/SafeComWrapper.cs

```diff
@@ -9,6 +9,9 @@ namespace Rubberduck.VBEditor.SafeComWrappers
     public abstract class SafeComWrapper<T> : ISafeComWrapper<T>
         where T : class
     {
+#if DEBUG
+        private const bool LogSuccessfulComRelease = false;
+#endif
```

Comment:
I believe this breaks the compilation for non-debug builds because if `DEBUG ` is false, the constant won't be defined but we reference it.

Reply:
The only use of the constant is in the other `DEBUG` only section further down.

Reply:
@bclothier even if it does, AppVeyor builds in Release, so no worries there